pkg/clusters/types/gke/vars.go

```diff
@@ -0,0 +1,25 @@
+package gke
+
+import (
+	"os"
+
+	"github.com/kong/kubernetes-testing-framework/pkg/clusters"
+)
+
+// -----------------------------------------------------------------------------
+// GKE Cluster - Vars
+// -----------------------------------------------------------------------------
+
+const (
+	// GKEClusterType indicates that the Kubernetes cluster was provisioned by Google Kubernetes Engine (GKE)
+	GKEClusterType clusters.Type = "gke"
+
+	// waitForClusterSeconds indicates the number of seconds to wait between cluster checks
+	// when deploying a new GKE cluster.
+	waitForClusterSeconds = 3
```

Comment:
optional but recommended: to me it's better to use `time.Duration` instead of plain numbers in places like this.
Also there's a `Duration` flag type present in `flag`/`pflag`

Reply:
resolved by 4243540a3d54213392015e49abbba187bb418291